site/zh/blog/new-in-devtools-93/index.md

```diff
@@ -0,0 +1,148 @@
+---
+layout: "layouts/blog-post.njk"
+title: "DevTools 新功能（Chrome 93）"
+authors:
+  - jecelynyeen
+date: 2021-07-28
+updated: 2021-07-28
+description:
+  "可编辑的 CSS 容器查询（container queries），web bundle 预览，更好地处理控制台里面的字符串以及更多。"
+hero: 'image/dPDCek3EhZgLQPGtEG3y0fTn4v82/mcT2bC9fEzrOLuVMOBlf.jpg'
+alt: ''
+tags:
+  - new-in-devtools
+  - devtools
+  - chrome-93
+---
+
+<!-- lint disable no-smart-quotes -->
+
+*感谢[流浪大法师](https://webfrontend.dev/)提供的翻译。*
+
+{% Aside %}
+有兴趣帮忙改进 DevTools 吗？请注册并参与 [Google 用户体验调研](https://google.qualtrics.com/jfe/form/SV_9YbKj35IGoGsDBj?reserved=1&utm_source=Website%20feature&Q_Language=zh&utm_medium=own_web&utm_campaign=Q4&productTag=chrm&campaignDate=November2020&referral_code=UXFm430458)。
+
+{% endAside %}
+
+
+## 样式窗格（Styles pane）可编辑 CSS 容器查询（container queries） {: #container-queries }
+现在，您可以在**样式**窗格（Styles pane）那里查看以及编辑 [CSS 容器查询（container queries）](https://web.dev/new-responsive/#responsive-to-the-container)。
+
+容器查询的出现，给响应式设计提供更多的动态解决方案。虽`@container` @规则的工作方式与媒体查询（`@media`）类似，然而 `@container` 查询的对象是满足一定条件的祖先容器节点，而不是查询视口以及 user agent 信息。
+
+在**元素**面板那里，点击带有 `@container` @规则的 DOM 元素。**样式**窗格现在将显示 `@container` 信息。您可以点击该信息，以编辑容器尺寸。**样式**窗格那里同时也会显示相对应的容器信息。鼠标 hover 上去，可以高亮页面中对应的容器元素以及检查容器的真实尺寸。点击可以选中该容器元素。
+
+容器查询特性现还处于实验性质。测试该特性之前，请在 `chrome://flags` 那里打开 `#enable-container-queries` 开关。
```

Comment:
I would suggest changing "实验性质" to "实验阶段", since "阶段" implies about a staged development process, while "性质" just focuses on container queries being "experimental" . It would also work better with "处于...".